exercises/D-logical-operators/exercise.js

```diff
@@ -5,20 +5,20 @@
   The variables should have values that match the expected results.
 */
 
-// Do not change these two statement
+// Do not change these two statements
 let htmlLevel = 8;
 let cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-let htmlLevelAbove5;
-let cssLevelAbove5;
+let htmlLevelAbove5 = (htmlLevel > 5);
+let cssLevelAbove5 = (cssLevel > 5);
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-let cssAndHtmlAbove5;
-let cssOrHtmlAbove5;
+let cssAndHtmlAbove5 = ((htmlLevel && cssLevel) > 5);
+let cssOrHtmlAbove5 = ((htmlLevel || cssLevel) > 5);
```

Comment:
Helena, please take another look. You cannot compare 'boolean' with numbers which is what you are doing here.
That is, If (htmlLevel && cssLevel) is **true** then you are asking is **true greater than 5** which is the incorrect question

Reply:
Yes, that's right. I can see it now.